claims_common.go

```diff
@@ -0,0 +1,89 @@
+package ccatoken
+
+import "fmt"
+
+const (
+	MaxLenRealmExtendedMeas = 4
+)
+
+func isCcaHashType(b []byte) error {
+	l := len(b)
+
+	if l != 64 {
+		return fmt.Errorf(
+			"%w: length %d (cca-hash-type MUST be 64 bytes)",
+			ErrWrongClaimSyntax, l,
+		)
+	}
+
+	return nil
+}
+
+func isValidChallenge(v []byte) error {
+	if err := isCcaHashType(v); err != nil {
+		return err
+	}
+
+	return nil
+}
+
+func isValidPersonalizationValue(b []byte) error {
+	l := len(b)
+
+	if l != 64 {
+		return fmt.Errorf(
+			"%w: length %d (cca-personalization-value MUST be 64 bytes)",
+			ErrWrongClaimSyntax, l,
+		)
+	}
+	return nil
+}
+
+func isValidRealmPubKey(b []byte) error {
+	// Realm Public Key is ECC Public key of type ECC-P384 of size 97 bytes
+	// In future things will change
```

Comment:
The 'in future bit' is rather vague. 
I suggest you remove this comment and add something at the top of the file along the lines of...
This code provides verification for the attestation claims as defined in the Realm Management Monitor
specification Beta-0 Document (DEN0137)
Then if anyone has any queries about the status of any of the claims, they can go and look them up in the spec.

Reply:
Thanks, I have removed the futuristic line. 
The top level README will indicate the referred specification!